rules/game_process.tex

```diff
@@ -75,6 +78,8 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee must tell the GameController operator to switch to \texttt{initial} state once everyone is in place or that teams are falling under forfeit rule (see \cref{sec:forfeit}).
```

Comment:
"The referee will announce the transition from the \texttt{setup} to \texttt{initial} ...."

Make the sentence simpler and consistent with the following text 2 sentences later.